pytition/pytition/settings/base.py

```diff
@@ -275,3 +276,6 @@
 MAINTENANCE_MODE_IGNORE_ADMIN_SITE = True
 # Following setting prevents error 500, due to issue #76 of django_maintenance_mode
 MAINTENANCE_MODE_STATE_FILE_PATH = os.path.join(BASE_DIR, 'maintenance_mode_state.txt')
+
+# Set default region to France in order to accept phone numbers without specifying prefix (+33)
+PHONENUMBER_DEFAULT_REGION = "FR"
```

Comment:
I am unsure if we should keep FR as default. Maybe edit the documentation to explain this setting (https://github.com/stefanfoulis/django-phonenumber-field#phonenumber_default_region)

Reply:
I guess it's OK like this, I can't see a better value for this setting. No perfect value here!